app/assets/javascripts/discourse/app/templates/composer.hbs

```diff
@@ -108,7 +108,8 @@
 
         </div>
 
-        {{composer-editor topic=topic
+        {{component composerComponent
```

Comment:
we should check of themes are not overriding this template

Reply:
discourse-gartner and discourse-basic-editor are both overriding it but this is not a concern of mine at the moment, they are outliers, and this is only done so I can test uppy in the composer using a site setting (because it is a mixin and I can't switch mixins based on a site setting AFAICT).

Reply:
Once uppy has been tested this line will be removed and we will just be using regular composer-editor once more.